pkg/test/resourcefixture/testdata/basic/secretmanager/v1beta1/secretmanagersecretversion/_generated_object_secretmanagersecretversion.golden.yaml

```diff
@@ -3,17 +3,20 @@ kind: SecretManagerSecretVersion
 metadata:
   annotations:
     cnrm.cloud.google.com/management-conflict-prevention-policy: none
+    cnrm.cloud.google.com/observed-secret-versions: (removed)
```

Comment:
This is a legacy annotation. The TF-based approach builds a mechanism  around this annotation to determine if a sensitive data is changed.